deploying.md

```diff
@@ -0,0 +1,113 @@
+# Deploying Philanthropy Data Commons software
+
+This document guides one through deployment for the back-end services at
+https://api.philanthropydatacommons.org and
+https://auth.philanthropydatacommons.org.
+
+The front-end at https://pilot.philanthropydatacommons.org has its own process.
+
+What gets deployed? A [docker compose yaml file][1].
```

Comment:
I didn't understand this sentence.  What gets deployed is the service itself, no?  A Docker compose YAML file is just part of the means by which that happens -- that file is a specification for the deployment, but that file is not itself being deployed.